GomokuServer/src/GomokuServer.Application/Games/Commands/Abstract/CreateGameForPair.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace GomokuServer.Application.Games.Commands.Abstract;
+
+public record CreateGameForPairCommand : ICommand<CreateGameResponse>
+{
+	[Required] public required int BoardSize { get; init; }
+	public required TimeControlDto TimeControl { get; init; }
+	public bool Anonymous { get; init; }
+
+	public required string FirstPlayerId { get; init; }
+	public required string SecondPlayerId { get; init; }
+}
+
+public class CreateGameForPairCommandHandler(
+	IDateTimeProvider _dateTimeProvider,
+	IAnonymousGamesRepository _anonymousGamesRepository,
+	IRegisteredGamesRepository _registeredGamesRepository)
+	: ICommandHandler<CreateGameForPairCommand, CreateGameResponse>
+{
+	private const int BoardMinSize = 13;
+	private const int BoardMaxSize = 19;
+
+	public virtual async Task<Result<CreateGameResponse>> Handle(CreateGameForPairCommand request, CancellationToken cancellationToken)
+	{
+		if (request.BoardSize < BoardMinSize || request.BoardSize > BoardMaxSize)
+		{
+			return Result.Invalid(
+				new ValidationError($"Board size cannot be less than {BoardMinSize} and more than {BoardMaxSize}"));
+		}
+
+		var gameSettings = new GameWithTimeControlSettings()
+		{
+			BoardSize = request.BoardSize,
+			TimeControl = new TimeControl(request.TimeControl.InitialTimeInSeconds,
+				request.TimeControl.IncrementPerMove)
+		};
+
+		var players = new Players(
+			new Player(request.FirstPlayerId, request.FirstPlayerId, TileColor.White),
```

Comment:
Second parameter in Player constructor is username. First Players constructor parameter should be black